src/common/Exd/ExdDataGenerated.cpp

```diff
@@ -2745,6 +2745,8 @@ Core::Data::HousingLandSet::HousingLandSet( uint32_t row_id, Core::Data::ExdData
   auto row = exdData->m_HousingLandSetDat.get_row( row_id );
   for ( int i = 0; i < 60; i++ )
     sizes.push_back( exdData->getField< uint8_t >( row, i ) );
+  for ( int i = 60; i < 60 + 60; i++ )
+    minPrices.push_back( exdData->getField< uint8_t >( row, i ) );
```

Comment:
min prices are certainly not uint8, if that was the case they would be 255 at the most ;)
they probably are uint32